techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf

```diff
@@ -88,8 +88,7 @@ bundle agent propagate_promises(trackingkey, technique_name, relay_sync_method,
                                                   "could_not_propagate_masterfiles",
                                                   "could_not_propagate_ncf_local_promise",
                                                   "could_not_propagate_ncf_common_promise",
-                                                  "nodeslist_error",
-                                                  "could_not_copy_license"
```

Comment:
I don't see the removal of this bundle

Reply:
https://github.com/Normation/rudder-techniques/pull/1719/files#r692873535